core/workflow-core/src/test/scala/edu/uci/ics/amber/storage/result/iceberg/IcebergDocumentConsoleMessagesSpec.scala

```diff
@@ -0,0 +1,46 @@
+package edu.uci.ics.amber.storage.result.iceberg
```

Comment:
For the test, you can simply mimic `IcebergDocumentSpec`: inherit `VirtualDocumentSpec` and override the generateSampleItems func